### PR TITLE
Image updates

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Related to my comment on the Dark Mode pull request you have.

### CODE REVIEW

File: static/images/company-project-logos/postgresql@2x.png
Comment (dscho) [inline static/images/company-project-logos/postgresql@2x.png:1]: This one seems to have lost the upper left corner of the "P"...

**Review by dscho — CHANGES_REQUESTED** [review]
Awesome work!

Could you describe your process, though? Maybe part of it is automatable.

In addition, I noticed that some details are lost, for example the rays in the Eclipse logo.

**Comment by TheGiraffe3** [comment]
I was using [GIMP](https://www.gimp.org), so I don't think it's automatable.

**Comment by dscho** [comment]
> I was using [GIMP](https://www.gimp.org), so I don't think it's automatable.

Fair enough.

Considering in particular the PostgreSQL logo in [the screenshot](https://github.com/git/git-scm.com/pull/1888#issuecomment-2510656771), what would you think about [the idea to always use a light background for those logos, even in Dark Mode](https://github.com/git/git-scm.com/pull/1888#issuecomment-2508883708), instead?

**Comment by TheGiraffe3** [comment]
That could work, but from what little I know of GIMP it would be quite a bit more difficult to add backgrounds then remove them.
I can try, but I'm not sure it'll work well (or look good on the website when that's in dark mode).

**Comment by TheGiraffe3** [comment]
Been trying: the Eclipse logo is pretty hard to effectively remove the background of while keeping all the rays.
Would it be allowable to get a different logo to remove the background of?

**Comment by dscho** [comment]
> Been trying: the Eclipse logo is pretty hard to effectively remove the background of while keeping all the rays.
> Would it be allowable to get a different logo to remove the background of?

I guess that Eclipse actually changed their logo in 2014, removing those rays. At least that's how it would appear when looking at https://www.eclipse.org/org/artwork/. If you download the IDE-related logo archive, you actually get an SVG, which should be a lot easier to work with than a PNG.

And on the IDE's homepage, they use https://eclipseide.org/images/logo/eclipse-ide-logo-white.svg... which we could theoretically use as-is.

File: static/images/company-project-logos/eclipse@2x.png
Comment (dscho) [inline static/images/company-project-logos/eclipse@2x.png:1]: This looks great!

For the record, it seems that the Git homepage already missed [an important update of Eclipse's logo in 2010 "to help broaden the appeal of the Eclipse to a younger demographic"](https://web.archive.org/web/20100404070643/http://www.eclipse.org/org/press-release/20100401_newlogo.php).

But the latest update of that logo happened [on Feb 13, 2014](https://blogs.eclipse.org/post/anonymous/introducing-updated-eclipse-logo) and is indeed reflected by this here commit.

**Review by dscho — APPROVED** [review]
Thank you so much!

**Comment by TheGiraffe3** [comment]
Happy to help.

**Pull request merged**